src/Umbraco.Core/Migrations/MigrationBase_Extra.cs

```diff
@@ -84,10 +85,18 @@ protected void AlterColumn<T>(string tableName, string columnName)
 
         protected void ReplaceColumn<T>(string tableName, string currentName, string newName)
         {
-            AddColumn<T>(tableName, newName, out var sqls);
-            Execute.Sql($"UPDATE {SqlSyntax.GetQuotedTableName(tableName)} SET {SqlSyntax.GetQuotedColumnName(newName)}={SqlSyntax.GetQuotedColumnName(currentName)}").Do();
-            foreach (var sql in sqls) Execute.Sql(sql).Do();
-            Delete.Column(currentName).FromTable(tableName).Do();
+            if (DatabaseType.IsSqlCe())
+            {
+                AddColumn<T>(tableName, newName, out var sqls);
+                Execute.Sql($"UPDATE {SqlSyntax.GetQuotedTableName(tableName)} SET {SqlSyntax.GetQuotedColumnName(newName)}={SqlSyntax.GetQuotedColumnName(currentName)}").Do();
+                foreach (var sql in sqls) Execute.Sql(sql).Do();
+                Delete.Column(currentName).FromTable(tableName).Do();
+            }
+            else
+            {
+                Execute.Sql(SqlSyntax.FormatColumnRename(tableName, currentName, newName)).Do();
```

Comment:
Wow, learn something new everyday. I didn't realize we had a FormatColumnRename on sql syntax and already had sp_rename on there. It's not actually used anywhere but nice to know we support the Migration syntax `Rename.Column(....).To(....)`